### ISSUE TITLE
More editing

### ISSUE THREAD

**Issue by NathanNovak** [StraboSpot/StraboSpot2#629]
<div type='discussions-op-text'>

<sup>Originally posted by **panamint55** October 14, 2019</sup>
On Strabo2 it would be great to be able to split spots into two spots.  I am not sure what the bahavior would be as far as splitting the ID, we will have to think about this.

Also, using an apple pencil, it might be nice to be able to "redraw" a spot from beginning to end.  I do a lot of contacts, and reshaping in this way would be great.

Not sure if this is an issue or feature request.  </div>

#238 similar

**Comment by jessicagood** [comment]
Added ability to split lines with commit 9e339314bc3a03bf636e5d23e56cd4972f75fbd5

**Comment by jessicagood** [comment]
Yeah, it doesn't have "redrawing" a from beginning to end, but I think it could be closed.
